### PR TITLE
Refold from caseygoodhew/grunt-cucumber

### PR DESCRIPTION
**Pull request by caseygoodhew**
- Updated README.md
- Removed use strict from process handler (strictly speaking, it's implied in modules)
- cucumberjs 
-- Applied suggested fixes from ESLINT
-- Comment inline about potential breaking change in failed test handling

### CODE REVIEW

File: tasks/cucumber.js [hunk 1]
```diff
@@ -184,10 +184,8 @@ module.exports = function(grunt) {
         handler(grunt, options, commands, function handlerCallback(err) {
             if (err) {
                 grunt.log.error('failed tests, please see the output');
```
Comment (caseygoodhew) [inline tasks/cucumber.js]: Invoking done(false) causes the grunt task to report a failure, which probably isn't the desired intention here - failed tests should not result in a failed grunt task.

The issue with this is that any subsequent grunt tasks do not run. In our case, our grunt command sequence looked like: 
```['mockserver_start', 'cucumber', 'mockserver_stop']```

In this case, if a test case failed, mockserver_stop was never invoked.

The expected behavior is that the task should succeed and failures should be dealt with by inspecting the test results in a subsequent step / process.

**Comment by caseygoodhew** [comment]
@AndrewKeig  @gkushang

**Comment by mavdi** [comment]
Looks good. I apologise, I don't get notifications from this repo for whatever crazy reason.

**Comment by caseygoodhew** [comment]
Thanks!

**Pull request merged**